### PR TITLE
Remove use of legacy context

### PR DESCRIPTION
**Pull request by jdhuntington**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8787)

### CODE REVIEW

**Comment by jdhuntington** [comment]
#7364 #7366

File: packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx [hunk 1]
```diff
@@ -211,6 +207,8 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     };
   }
 
+  private _getContext = (): IScrollablePaneContext => this.context;
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx]: Why is a separate getter needed here?

**Comment by ecraig12345** [comment]
Nice! @leddie24 IIRC you have some experience with ScrollablePane and Sticky, which is what this change touches--if that's correct, would you mind taking a look? (or recommending someone?)

File: packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx (hunk 1 above)
Comment (jdhuntington) [reply packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx]: The types for react 16.8 don't give type safety on context. This getter seemed like the cleanest way to get the type annotation propagated.

Comment (ecraig12345) [reply packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx]: Well that's annoying :(

**Pull request merged**